iron-multi-selectable.html

```diff
@@ -77,6 +78,18 @@
         (this.selectedValues != null && this.selectedValues.length);
     },
 
+    _updateAttrForSelected: function() {
+      if (this._shouldUpdateSelection) {
+        if (this.multi) {
+          this.selectedValues = this.selectedItems.map(function(selectedItem) {
+            return this._indexToValue(this.indexOf(selectedItem));        
+          }, this);
+        } else {
+          this.selected = this._indexToValue(this.indexOf(this.selectedItem));        
```

Comment:
Maybe flip this so that it's
```
if !this.multi
  call super
else if this._shouldUpdateSelection
  do selectedValues mapping
```
.. then, the _shouldUpdateSelection test only happens once when multiple selection is disabled.